### DISCUSSION TITLE
Save cover images to files?

### DISCUSSION THREAD

**Discussion by AlfredTe** [enzo1982/freac#634] [General Discussion]
How does it work? Save cover images to files? I have ripped a CD but there are no cover images in the folder.
!

[Bildschirmfoto 2024-09-07 um 11 13 34](https://github.com/user-attachments/assets/92b0a29f-0171-4a62-8b79-d58793cf7d62)

**Comment by cleanyoursockets** [comment]
![freac](https://github.com/user-attachments/assets/5b85aa7a-921e-4e5f-b892-ba7f7f4404b6)
You have to add cover manually, insert a cd, press tags and and album art.
